test/cmake/seqan3_require_benchmark.cmake

```diff
@@ -0,0 +1,47 @@
+# -----------------------------------------------------------------------------------------------------
+# Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+# Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+# This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+# shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+# -----------------------------------------------------------------------------------------------------
+
+# Exposes the google-benchmark target `gbenchmark`.
+macro (seqan3_require_benchmark)
+    enable_testing ()
+
+    set (gbenchmark_project_args ${SEQAN3_EXTERNAL_PROJECT_CMAKE_ARGS})
+    list (APPEND gbenchmark_project_args "-DBENCHMARK_ENABLE_TESTING=false")
+    # list (APPEND gbenchmark_project_args "-DBENCHMARK_ENABLE_LTO=true")
```

Comment:
What is the purpose of this comment?

Reply:
It was there before.

The documentation for google benchmark suggest to use LTO, see https://github.com/google/benchmark#debug-vs-release.

We don't really need it, because we don't link any code, but it might be still interesting to use it. I think I had some problems using google benchmark with LTO, so I commented it out again, but kept it, because it might be handy.

But since I never touched this line ever again we might just remove it. :shrug:

Reply:
I would say either remove it or add a comment with the link, so it is clearer why it is there?

Reply:
Explained it :)